internal/store/postgres/migrations/000019_alter_asset_table_refreshed_at.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE assets ADD COLUMN IF NOT EXISTS refreshed_at TIMESTAMP DEFAULT NOW();
```

Comment:
Is it possible to not having default value?

Reply:
Yap, it is possible, to be Null as default

Reply:
After reconsideration, I agree that it is better to set as NULL ya as default, so we don't confused which one that already refreshed after alter table.